packages/framework-info/package.json

```diff
@@ -19,6 +19,7 @@
     "build:browser": "run-s build:browser:*",
     "build:browser:core": "vite build",
     "build:browser:site-root": "cpy index.html ./dist",
+    "pretest": "run-s prebuild",
```

Comment:
when running `npm test` locally ensure that the generated files are up2date. We do not need to run the whole build before the test, but only the file generation